### ISSUE TITLE
Incorrect Android developer ID and app ID set in Google Play Store

### ISSUE THREAD

**Issue by slook** [jackBonadies/SeekerAndroid#8]
- Android dev ID is "companyname" which is incorrect, because it's not the name of the "ShinjiIndustrial" developer name.

- Android app ID is "andriodapp1" which is incorrect, because it's not the name of the "Seeker" application name.

- It is difficult to determine this is the proper genuine app, because "jackBonadies" is not mentioned prominently enough in the store page description, suggest move the GitHub link above the scroll fold.

Consider creating a new G account for the "jackBonadies" developer to match the GitHub page, since this is a known reputable dev but the "ShinjiIndustrial" nickname is unknown. This would provide opportunity to implement both the enhancements suggested above.

Alternately, it might be better to create a new GitHub account for "ShinjiIndustrial", since there are many good reviews on Google, but not much in the repo.

See link:
https://play.google.com/store/apps/details?id=com.companyname.andriodapp1

**Comment by jackBonadies** [comment]
Yeah I wish I could change the dev ID and app ID.  It was my first time making an Android App.  I fear that there is not really a good solution to the issue described since I failed to set the ID properly and now I am stuck with it.

I can move up the GitHub link to above the fold.

> Consider creating a new G account for the "jackBonadies" developer to match the GitHub page
My fear with this is that wouldn't there then be two Seeker applications on the playstore?  And if I delete the original then all reviews will be lost and the users ability to update the app will be lost.

>new GitHub account for "ShinjiIndustrial"
I can do this.

**Comment by slook** [comment]
Yes I can understand your predicament, however I suggest it's better to take some action sooner rather than later, because it appears as if the app is going to be successful, but it will always be hindered by your initial error.

Perhaps the best you can do at this point is aim to develop version 3+ on the new GitHub account and release it as a separate application on your ShinjiIndustrial Google account, and try to encourage your existing users to migrate over to it somehow. This way, there will be a historic record of reviews specific to each major version branch by the same dev, which verifies it's authenticity.

**Comment by now0clock** [comment]
I came here about the same thing. I was looking at the apps installed on my device, and saw "com.companyname.andriodapp1" then thought wtf is this? I never noticed.

I recently tried to login and my PW wasn't saved like it always has been, and didn't find a backup, so I had to make a new username. Argh. (Had the app installed from Izzy's repo in F-Droid for quite a while now.)

I don't know really how it works with app name, package names, but Maybe what you could do about the app's name is give it a newer Name or something like what SLSK did with, Soulseek QT, or SLSK NS (new server). Like.. "Seeker NG" or something..? Seeker LX XZ etc.. lol. Pz.

**Comment by slook** [comment]
It is a good idea, but perhaps it would be better to rename the 2.x package to something like "Seeker Classic", and then retain the "Seeker" name for new the 3.x branch with the correct package name on the stores.

I suppose "Seeker Classic" would eventually become the version that legacy users would need for running it on older versions of Android, so having it active on the Play store would still serve some useful purpose.

**Comment by now0clock** [comment]
I agree, this would work also. Either way, it needs a proper package name asap.

Some ideas:
Maybe the new, properly named app can just reside on Github for Izzy's repo, and not be in the Play Store at all. (Fork the GPlay version into a new GH account, etc) Then you could resubmit the new app for inclusion in F-Droid main repo..?

I am pretty sure most people getting their apps from GPlay don't care about things like Dev ID, App ID, package name, or even the Source Code of the app at all. It would be great to see this in F-Droid as a properly named FOSS application.
